### PR TITLE
feat(observability): implements client side logging configuration

### PR DESCRIPTION
**Pull request by ygrishajev**
Stats-web uses experimental app dir. It hydrates client side in a certain way. To make sure logger is instantiated with proper config it should be configured under `use client` directive. Further more, logger instantiated in module scope (outside react context) would actually be instantiated on the backend and would ignore client side config. So to make sure it works, it should be in the context. For this a `useLogger` hook is added.

### CODE REVIEW

**Comment by ygrishajev** [comment]
@forbesus FYI

**Comment by forbesus** [comment]
> @forbesus FYI

https://github.com/akash-network/console/pull/462#discussion_r1863468758
how about this?

**Comment by forbesus** [comment]
we can use logger without using custom hook and changing next configuration, i think @ygrishajev

**Comment by ygrishajev** [comment]
> we can use logger without using custom hook and changing next configuration, i think @ygrishajev 

Demo?

File: package-lock.json
Comment (baktun14) [inline package-lock.json:1]: Something wrong with the lock file

**Review by baktun14 — APPROVED** [review]

**Pull request merged**